docs/02-app/01-building-your-application/06-configuring/02-eslint.mdx

```diff
@@ -202,6 +202,21 @@ The `next/core-web-vitals` rule set is enabled when `next lint` is run for the f
 
 > The `next/core-web-vitals` entry point is automatically included for new applications built with [Create Next App](/docs/app/api-reference/create-next-app).
 
+### TypeScript
+
+In addition to the Next.js ESLint rules, `create-next-app --typescript` will also add [`@typescript-eslint`'s base recommended rules] to your config:
```

Comment:
What are the square brackets here for? Is there a missing link?

Reply:
D'oh, thanks - yes, I ended up moving that link to after the code snippet.